src/sentry/sentry_metrics/querying/api.py

```diff
@@ -49,6 +60,21 @@ class MetricsQueryExecutionError(Exception):
 GroupKey = Tuple[Tuple[str, str], ...]
 
 
+@dataclass(frozen=True)
+class ExecutableQuery:
+    query: MetricsQuery
```

Comment:
nit: can we call this `metrics_query`. We anyways overload the word query and the accesses like `self.query.query.query` are really confusing

Reply:
I was thinking about having a method that gives us access to it directly.